index.js

```diff
@@ -59,7 +59,7 @@ client.on('message', async msg => {
 		index = online_mentor_afk_list.findIndex((afk_mentor) => msg.author === afk_mentor.name)
 		if(index !== -1) {
 			online_mentor_afk_list.splice(index, 1)
-			msg.reply("has returned to their keyboard. They will be right with you :hugging::raised_hands:")
+			msg.channel.send("has returned to their keyboard. They will be right with you :hugging::raised_hands:")
```

Comment:
```suggestion
			msg.channel.send(`${msg.author} has returned to their keyboard. They will be right with you :hugging::raised_hands:`)
```

Might want to test out what the message looks like 😄

Reply:
Awesome, that does work - I'll swap everything over to that syntax as it's a lot cleaner